kafl_fuzzer/common/config.py

```diff
@@ -15,12 +15,13 @@
 from kafl_fuzzer.common.logger import logger
 
 
-
-
 class FullPath(argparse.Action):
     def __call__(self, parser, namespace, values, option_string=None):
         setattr(namespace, self.dest, os.path.abspath(os.path.expanduser(values)))
 
+def debug_print(message, *args):
+    if 'KAFL_CONFIG_DEBUG' in os.environ:
+        print(f"[CONFIG] {message}" % args)
```

Comment:
`logger.info()` ?

Reply:
logger is not initialized here, in particular the --debug flag is not parsed yet.